File: backend/src/main/java/com/example/skilltracker/stats/OverviewStatsDto.java
package com.example.skilltracker.stats;

import java.util.List;

public record OverviewStatsDto(long totalMinutes,
                               long skillsCount,
                               long sessionsCount,
                               List<OverviewSkillBreakdownDto> bySkill,
                               List<InactiveSkillDto> inactiveSkills) {
}

File: android/app/src/main/java/com/example/skilltracker/data/dto/OverviewStatsDto.kt
package com.example.skilltracker.data.dto

data class OverviewStatsDto(
    val totalMinutes: Long,
    val skillsCount: Long,
    val sessionsCount: Long,
    val bySkill: List<OverviewSkillBreakdownDto>,
    val inactiveSkills: List<InactiveSkillDto>
)
